### DISCUSSION TITLE
js proposal

### DISCUSSION THREAD

**Discussion by rinka88** [templaza/astroid-framework#610] [Ideas]
In the js folders of the framework I saw the file slick.js, which is included in the widgets. Does it turn on itself when the widget is activated, and will not be available at other times? that is, we can’t use it in other places? for example for a separate module, in overrides? And why did the choice fall on slick.js, it seems that it has not been updated for a long time. Of the current ones, bootstrap or swiper.js itself seems to be updated normally.

**Comment by sonvnn** [comment]
@rinka88 Slick is only called in widgets that require it to be used. In other places it will not be called. Currently the Slick meets some requirements that Bootstrap cannot. That's why it's used.
